common/src/main/java/com/viaversion/viaversion/util/Config.java

```diff
@@ -207,4 +212,15 @@ public double getDouble(String key, double def) {
             return new ArrayList<>();
         }
     }
+
+    public @Nullable JsonElement getSerializedComponent(String key) {
+        final Object o = this.config.get(key);
+        if (o != null && !((String) o).isEmpty()) {
+            return GSON_SERIALIZER.serializeToTree(Component.text((String) o)
```

Comment:
Use the legacy section deserializer and directly serialize again without the rest

Reply:
May I know what the "legacy section deserializer" means? [this?](https://github.com/ViaVersion/ViaVersion/blob/master/common/src/main/java/com/viaversion/viaversion/protocols/protocol1_13to1_12_2/ChatRewriter.java#L32)

Reply:
`LegacyComponentSerializer.legacySection()`

Reply:
it's Adventure serializer for legacy strings "§cHello §bWorld"  : LegacyComponentSerializer. legacySection()

Reply:
Thank you! I just pushed the commit.